cmd/action/apply.go

```diff
@@ -31,16 +32,18 @@ var (
 	ApplyCmd = &cobra.Command{
 		Use:   "apply",
 		Short: "Apply an atlas schema to a target database.",
-		Long: "`atlas schema apply`" + ` plans and executes a database migration to bring a given database
-to the state described in the Atlas schema file. Before running the migration, Atlas will print the migration
-plan and prompt the user for approval.
+		// Use 80-columns as max width.
```

Comment:
:-1: I don't like 80 cols. We are no longer working on SD resolutions and no-one is coding on a smartphone

Reply:
What do you don't like about this output? When the screen size is smaller (my case), the line breaks make it ugly. 

![image](https://user-images.githubusercontent.com/7413593/156360332-b8eb6e8d-8a7f-4c64-98a6-00c952e445c7.png)

Reply:
Your screen supports 120 too by the looks of it.

Reply:
![image](https://user-images.githubusercontent.com/12862103/156361004-2ce088c9-7f01-4da3-b937-327620795ae8.png)